docs/_docs/best-practices/compute/compute.md

```diff
@@ -88,26 +76,60 @@ resources
 | project id, PowerState, VMLocation, resourceGroup, subscriptionId
 ```
 
-### Query: Virtual machine scale set details
+<br>
 
-This query analyzes Virtual Machine Scale Sets (VMSS) in your Azure environment based on their SKU, spot VM priority, and priority mix policy. It provides insights for cost optimization and resource management strategies.
+### Query: List Virtual Machines deallocated
+
+This Azure Resource Graph (ARG) query identifies Virtual Machines (VMs) in your Azure environment that are in the 'deallocated' state. It retrieves details about their power state, location, resource group, and subscription ID.
 
 <h4>Category</h4>
 
-Resource management
+Waste reduction
 
 <h4>Query</h4>
 
 ```kql
 resources
-| where type =~ 'microsoft.compute/virtualmachinescalesets'
-| extend SpotVMs = tostring(properties.virtualMachineProfile.priority)
-| extend SpotPriorityMix = tostring(properties.priorityMixPolicy)
-| extend SKU = tostring(sku.name)
-| extend resourceGroup = strcat('/subscriptions/', subscriptionId, '/resourceGroups/', resourceGroup)
-| project id, SKU, SpotVMs, SpotPriorityMix, subscriptionId, resourceGroup, location
+| where type =~ 'microsoft.compute/virtualmachines'
+    and tostring(properties.extended.instanceView.powerState.displayStatus) == 'VM deallocated'
+| where resourceGroup in ({ResourceGroup})
```

Comment:
What is `{ResourceGroup}` here?

Reply:
This is used by the workbook to filter the results based on a parameter called ResourceGroup. We can remove it from this query.